schema.json

```diff
@@ -379,19 +379,44 @@
                         }
                       }
                     }
-                  },
-                  "supported_access_modes": {
-                    "description": "Supported access modes for this entity",
-                    "type": "array",
-                    "items": {
-                      "type": "string"
-                    }
                   }
                 },
                 "required": ["traits"],
                 "additionalProperties": false
               }
             },
+            "exceptions": {
```

Comment:
[Consider] I know this is copying an existing word, but this word can be interpreted in a bunch of different ways. Putting aside the exceptions meaning errors issue, maybe even just `exceptionsByAccess` or something might be a good pattern to establish? To at least indicate the branching axis.

Reply:
I agree, it rubs me the wrong way too. If we're going to change the word, perhaps `overrideByAccess`, rather than using "exception"?

Reply:
Override is better in every sense, i'd be fine if you wanted to change that just here and log an issue to change up the rest, (although tbh we should prioritise it because the inconsistency would be hella confusing)

Reply:
I went through all our repos - none of them use the `"exceptions"` feature of `"managementPolicy"`.  

Any other downstream repos will surely not be installing from `main`, so updating won't break them immediately.

Downstream consumers may break after a release, which I think is fine since we're still in alpha and can call out the breaking change in the release notes.

So let's just update the wording for both.

Reply:
Renamed to `"overrideByAccess"` in 66e2b679 then renamed `managementPolicy` `"exceptions"` to `"overrideByTraitSet"` in 144c8c59.

I still need to re-word the commit message, will do when rebasing.